app/Parser.php

```diff
@@ -1051,7 +1051,7 @@ public function validate_license($license)
 			'without permission',
 			'without prior auth',
 			'you may not',
-			'Proprietery',
+			'Proprietary',
```

Comment:
https://github.com/WordPress/wordpress.org/blob/9209afdd3f0373f99f473f28d5931e2288c9c92a/wordpress.org/public_html/wp-content/plugins/plugin-directory/readme/class-parser.php#L972 👀

Reply:
This is probably intended.